pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -659,29 +661,74 @@ type InterfaceTarget struct {
 }
 
 type Alias struct {
+	name        string
+	userDefined bool
+}
+
+// Package private, responsible to interact with xml.Encoder and xml.Decoder
+type userAliasXML struct {
 	Name string `xml:"name,attr"`
 }
 
-type UserAlias Alias
+// Package private, responsible to interact with json.Marshal and json.Unmarshal
+type userAliasJSON struct {
+	Name        string
+	UserDefined bool
+}
 
 type Rom struct {
 	Enabled string `xml:"enabled,attr"`
 }
 
+func NewLibvirtManagedAlias(aliasName string) *Alias {
```

Comment:
This is only used in the unit tests for this package, can you make it `newLibvirtManagedAlias`?

Reply:
If it's only used in unit tests, can it be defined in schema_test.go?

Reply:
Moved the func-def into schema_test.go and kept it package-private.